doc/sphinx/developer_manual/developing_for_the_GWB/creating_new_plugins.md

```diff
@@ -2,6 +2,17 @@
 Creating new plugins
 ====================
 
-```{todo}
-Explain how to create a new plugin. If possible find one which already nearly does what you want and copy and change it, but also explain how to build one from sratch.
-```
\ No newline at end of file
+It is possible that the user wants to implement a different distribution of properties (temperature or composition) than the ones available in the GWB. This can be achieved by creating a new plugin with the steps outlined subsequently. We will discuss two ways to write a plugin: one is to base it on an existing plugin, and another is to write the plugin from scratch.
+
+#### Modify an existing plugin
```

Comment:
You need to use ## to make readthedocs happy (you always start with # as the main heading and cannot reduce the heading level by more than one for each heading).